src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
     <metadata>
         <id>Azure.DCAP.Windows</id>
-        <version>0.0.2</version>
+        <version>0.0.3</version>
         <!-- Authors contain text that appears directly on the gallery -->
         <authors>Microsoft</authors>
         <owners>Microsoft</owners>
```

Comment:
Can you also update releaseNotes?

Reply:
Sorry, I am not clear which file contains the release notes.  Which file are you referring to?

Reply:
Within the same .nuspec file there is a releasenotes section were you should elaborate on the changes made to this release of the Azure DCAP for Windows. For ex: Exporting function calls for QEID querying etc.